### PR TITLE
fix(select): options width and selected-value ellipsis

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/select/README.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Read more about [vwc-option](../../components/option).
   }
 </style>
 <vwc-select label="choose one option">
-  <vwc-option value="1" text="Option 1" selected></vwc-option>
-  <vwc-option value="2" text="Option 2"></vwc-option>
-  <vwc-option value="3" text="Option 3"></vwc-option>
+  <vwc-option value="1" text="Option 1: dogs" selected></vwc-option>
+  <vwc-option value="2" text="Option 2: cats"></vwc-option>
+  <vwc-option value="3" text="Option 3: doges and catas"></vwc-option>
```
Comment (rachelbt) [inline libs/components/src/lib/select/README.md:26]:
```suggestion
  <vwc-option value="3" text="Option 3: dogs and cats"></vwc-option>
```

**Review by rinaok — APPROVED** [review]

File: libs/components/src/lib/select/README.md [hunk 2]
```diff
@@ -21,9 +21,9 @@ Read more about [vwc-option](../../components/option).
   }
 </style>
 <vwc-select label="choose one option">
-  <vwc-option value="1" text="Option 1" selected></vwc-option>
-  <vwc-option value="2" text="Option 2"></vwc-option>
-  <vwc-option value="3" text="Option 3"></vwc-option>
+  <vwc-option value="1" text="Option 1: dogs" selected></vwc-option>
```
Comment (YonatanKra) [inline libs/components/src/lib/select/README.md:24]: Why the change?

**Review by YonatanKra — APPROVED** [review]

File: libs/components/src/lib/select/README.md (hunk 2 above)
Comment (rachelbt) [reply libs/components/src/lib/select/README.md:24]: for seeing a better diff in text-width when selected

Comment (rinaok) [reply libs/components/src/lib/select/README.md:24]: @rachelbt I can't see this dogs text in the snapshots
